schemas/com.snowplowanalytics.mobile/deep_link/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Entity that indicates a deep-link has been received and processed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "deep_link",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"url": {
+			"type": "string",
+			"description": "URL in the received deep-link",
+			"maxLength": 255
```

Comment:
That's quite short for a URL. I'd suggest upping it to 4096 (same for referrer too).